pom.xml

```diff
@@ -136,162 +136,6 @@
   </build>
 
   <profiles>
-    <profile>
-      <id>latest-sdk</id>
```

Comment:
could you tell me where use `latest-sdk` profile, didn't find it. thanks

Reply:
Originally, it's used manually when running in command line like `mvn clean package -P latest-sdk`.